ckan/model/user.py

```diff
@@ -59,7 +59,8 @@ def set_api_key() -> Optional[str]:
         Column('sysadmin', types.Boolean, default=False),
         Column('state', types.UnicodeText, default=core.State.ACTIVE),
         Column('image_url', types.UnicodeText),
-        Column('plugin_extras', MutableDict.as_mutable(JSONB))
+        Column('plugin_extras', MutableDict.as_mutable(JSONB)),
+        Column('plugin_data', MutableDict.as_mutable(JSONB))
```

Comment:
IMHO we should rename plugin_extras into `plugin_data` here. There's no reason to have two places to store namespaced plugin-specific data

Reply:
my understanding was that we would like to have separate space to store private and public data. 
so `plugin_extras` would be place to keep plugin related data and it would be available only for sysadmins 
and `plugin_data` would be stored in the user_extras table had we proceeded with that approach.